Automattic/wp-calypso
# i18n: Use wp-babel-makepot for translate script

#### Changes proposed in this Pull Request

* Replace `i18n-calypso` CLI with `wp-babel-makepot` for extracting translatable strings from JS source.

#### Testing instructions

* Run `yarn run translate`
* Confirm translatable strings are successfully extracted in `calypso-string.pot. Individual POT segments should be output in `./build/strings`
